src/main/java/com/gregtechceu/gtceu/common/machine/electric/FisherMachine.java

```diff
@@ -276,7 +276,7 @@ public boolean isOpenWaterFishing() {
                 useBait |= tryFillCache(itemStack);
 
             if (useBait)
-                this.baitHandler.extractItem(0, 1, false);
+                this.baitHandler.getStackInSlot(0).shrink(1);
```

Comment:
uh don't do this, you could do `this.baitHandler.storage.extractItem(0, 1, false)` instead, but don't call `shrink` on a stack in a handler, that's forbidden and documented as so

Reply:
Done. Also for future reference, why not use shrink?